config/initializers/active_support_extensions.rb

```diff
@@ -0,0 +1,30 @@
+#this makes sure that hash with indifferent access gets deep converted into hash.
```

Comment:
http://www.bonkersworld.net/images/2012.08.15_todo.png Also copying a code from somewhere else without attribution is not the best practice (I'm actually not sure about the legal issues and licenses, although from GPL -> MIT it would probably be worse)